app/tests/compilers/css.js

```diff
@@ -0,0 +1,4 @@
+function noop() { return null; }
+
+require.extensions['.css'] = noop;
```

Comment:
Wouldn't babel convert it into commonJS with our [preset](https://babeljs.io/docs/plugins/preset-es2015/) and from there NodeJS will just require it?

I put a console log in one of our jsx files that are tested and it was logged.